### ISSUE TITLE
Freeze on Joining Server

### ISSUE THREAD

**Issue by CorkGogo2010** [DaeFennek/libpega#4]
When I load into my server, I can play for a few seconds, then it freezes, and I have to hard shut down the Wii. It does this both when loading the app on the Wii directly and when launching through wiiload. I am using your provided server.properties. I also had another person try to join and they had the same issue. Also, the issue with the default username is still present. Attached is a video of me launching it, my server log, and a picture of an error code when I try to shut down the Wii (may be unrelated to the freeze). 
[server.log](https://github.com/kperdlich/wiicraft/files/4241630/server.log)
[Wiicraft_Crash.zip](https://github.com/kperdlich/wiicraft/files/4241644/Wiicraft_Crash.zip)

**Comment by iyenal** [comment]
Unfortunately having same issue, even if trying multiple times.
However it can be launched on Wii directly using WiiXplorer without wiiload (HBC being random) even if the same freeze occurs. Thanks in advance.

**Comment by kperdlich** [comment]
The playername should now finally be fixed. There is a problem in certain bioms especially jungle. Can you check in which bioms it happend for you?

**Comment by CorkGogo2010** [comment]
Yes it was the Jungle biome, in the server.properties, you had set the seed to Wiicraft, which happens to spawn you in the Jungle.

**Comment by kperdlich** [comment]
Hm I never spawned in the jungle with the Wiicraft seed but the jungle is definitley the problem.

**Comment by WiseMatthack** [comment]
Great to know. Now we might be able to actually enjoy Wiicraft!

**Comment by GitGramm** [comment]
I have the same issue with freezing on startup the jungle biome. Is there a possible workaround? Thanks!

**Comment by CorkGogo2010** [comment]
Just change the server seed in server.properties

**Comment by GitGramm** [comment]
It works! Thanks!

**Comment by kdude729** [comment]
what biome do you have to spawn in because i set the seed to plains and it still crashed
